doc/nrf/gs_recommended_versions.rst

```diff
@@ -0,0 +1,23 @@
+.. _gs_recommended_versions:
+
+Required Tools
+##############
+
+The following table show the list of tools required for working with |NCS| **v1.3.0**.
```

Comment:
```suggestion
The following table shows the tools that are required for working with |NCS| **v1.3.0**.
```

Reply:
Can we extract the NCS version automatically here? Since this is now for v1.3.99 already, not v1.3.0.

Reply:
done